### ISSUE TITLE
--instances=-1 stop working

### ISSUE THREAD

**Issue by maksimlikharev** [Unitech/pm2#2953]
**What's going wrong?**
according to docs `-1 to spread the app across all CPUs - 1`, so previously if # of CPU 1 it will use 1,
in 2.5.0 pm2 stop starting the service.

**How could we reproduce this issue?**
pm2 --instances=-1 your-script-runner.js

make sure you have 1 CPU instance

## Supporting information

```
PM2 version: 2.5.0
Node version: v6.9.1
EC2 Amazon Linux t2.small
```

**Comment by maksimlikharev** [comment]
if i'm looking at the correct version of the code, it used to be smarter than that:
```
env.instances = parseInt(env.instances);
  if (env.instances === 0) {
    env.instances = numCPUs;
  } else if (env.instances < 0) {
    env.instances += numCPUs;
  } else if (env.instances <= 0) {
    env.instances = 1;
  }
```
should it be ?
```
env.instances = parseInt(env.instances);
  if (env.instances === 0) {
    env.instances = numCPUs;
  } else if (env.instances < 0) {
    env.instances += numCPUs;
  } 

if (env.instances <= 0) {
    env.instances = 1;
  }
```

**Comment by vmarchaud** [comment]
Indeed its my bad i introduced this bug via https://github.com/Unitech/pm2/commit/35945e82d0a67c2368b4ab17facb96da612e0e2f, i'll make a patch quickly, thanks for the report.

**Comment by Unitech** [comment]
Published on PM2 2.6.0:

```bash
$ npm install pm2@latest -g
$ pm2 update
```
